code/sound/ffmpeg/WaveFile.cpp

```diff
@@ -169,14 +167,21 @@ bool WaveFile::Open(const char *pszFilename, bool keep_ext)
 				throw FFmpegException("Unknown file extension.");
```

Comment:
This error should probably be clarified (if the extension isn't specified, how can it be unknown?), but may be out of scope for this PR.